user/languages/clojure.md

```diff
@@ -41,7 +41,7 @@ Projects that find this sufficient can use a very minimalistic .travis.yml file:
 
     language: clojure
 
-### Using Midje on travis-ci.org
+### Using Midje or Speclj on travis-ci.org
```

Comment:
Is there a reason that this is an "or" heading here? Is Speclj any way related to Midje? If it is not, I feel that is is much cleaner to have a separate section instead.